baselines/disable-packing-test/src/v1beta1/echo_client.ts.baseline

```diff
@@ -607,6 +607,11 @@ export class EchoClient {
     this.initialize();
     return this.innerApiCalls.wait(request, options, callback);
   }
+  async waitDecodeLRO(name: string){
+    const [operation] = await this.operationsClient.getOperation({name}, {}) as unknown as [operationProtos.google.longrunning.Operation];
```

Comment:
All the casting happening in here makes me nervous.  what's up?

Reply:
I agree, this should be possible to fix (with some changes on the gax side).

@xiaozhenliu-gg5 I think the right fix here would be to upgrade `operationClient.ts` in gax to a proper TypeScript client with proper types (pretty much as you did with IAM client). Right now, `operationClient.ts` methods don't have return type declarations, so TypeScript infers the types from the method code and get much more general results than we need.

Let's make a gax PR that adds proper types to `operationsClient.ts` and then update this PR with those types.

Reply:
Sure, I will first fix types for operationClient and then update proper type here. Thanks!